floodlight/io/datasets.py

```diff
@@ -0,0 +1,84 @@
+import os
+import tempfile
+from typing import Tuple
+
+import h5py
+
+from floodlight.io.utils import extract_zip, down_loader
+from floodlight import XY, Pitch
+from settings import EIGD_HOST_URL, DATA_DIR, EIGD_FILE_EXT, EIGD_FRAMERATE
```

Comment:
I think ``EIDG_HOST_URL``, ``EIDG_FILE_EXT`` and ``EIGD_FRAMERATE`` should be refactored into class attributes as they will only be used in this scope